src/Google.Api.Gax.Grpc/Google.Api.Gax.Grpc.csproj

```diff
@@ -33,8 +33,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Grpc.Auth" Version="1.7.0" />
-    <PackageReference Include="Grpc.Core" Version="1.7.0" />
+    <PackageReference Include="Grpc.Auth" Version="1.7.1" />
```

Comment:
Grpc.Auth 1.7.1 doesn't exist. I'm not sure if it should or not!

Reply:
Ooh, indeed. Updated the PR.